tools/python/boutiques/schema/descriptor.schema.json

```diff
@@ -582,7 +598,7 @@
                 },
                 "additionalProperties": false
             },
-        "minItems": 1,
+        "minItems": 0,
```

Comment:
instead of allowing an empty array, why don't we just make `output-files` optional? That might break a few things so we should have a test for that.

Reply:
Agreed - updated the validator accordingly. This is tested in the tests for `bosh create`, since it now creates a descriptor without `output-files` from argparsers.